docs/finality-provider-operation.md

```diff
@@ -954,7 +967,49 @@ This will withdraw **ALL** accumulated rewards to the address you set in the
 `set-withdraw-addr` command if you set one. If no withdrawal address was set,
 the rewards will be withdrawn to your finality provider address.
 
-Congratulations! You have successfully set up and operated a finality provider.
+### 6.8 Refunding finality provider
+
+To support the gas costs associated with committing randomness, which are not
+refunded by the protocol, we recommend setting up a refunding flow.
+This involves periodically transferring rewards from a beneficiary key to the
+operational key used by the finality provider.
+
+**How to set up the refunding process:**
+
+1. Ensure you have two keys:
+   * Beneficiary Key: Receives staking rewards.
+   * Operational Key: Used by the finality provider daemon to submit transactions.
+
+  If this step hasnt been completed, follow the steps in
+  [5.2 Add key for the Babylon account](#52-add-key-for-the-babylon-account) and
+  create 2 additional keys.
+
+2. Configure Withdrawals:
+  To set the withdraw address to the beneficiary key, use the following command:
+
+  ```shell
+  fpd set-withdraw-addr <beneficiary-key> --from <registered-bbn-address>
+  --keyring-backend test --home <home-dir> --fees <fees>
+  ```
+
+  This sets the withdraw address to the beneficiary key to enable
+  withdrawing rewards periodically.
+
+3. Fund the Operational Key:
+  Lastly, set the operational key name in the keyring home directory in the
+  `[babylon]` config in `fpd.conf`.
```

Comment:
We need another step where the operator needs to create a cron job to periodically fund the operational key